_includes/team_list.html

```diff
@@ -17,7 +18,10 @@ <h3 class="team-item-title">
                   <span style="font-size:10px;">New</span>
                 {% endif %}
             </h3>
-            <p class="team-item-subtitle"><b>{{ one_page.role }}</b></p>
+            <p class="team-item-subtitle"><b>{{ one_page.role }}</b><br>
+            <small>{{ one_page.start_date }}{% if one_page.start_date %} - {% endif %}{{ one_page.end_date | default: t.Current[page.lang] }}</small>
```

Comment:
vedi mio commento su Paolo Barberis, sposterei l'`endif` alla fine cosi' se non c'è la `start_date` non appare nulla

Reply:
per non bloccare questa PR che serve per domani, la questione data di Paolo la sposto in un'altra issue